index.html

```diff
@@ -1,12 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <link rel="stylesheet " href="./styles/index.css">
+    <title>J&G Weather Report</title>
 </head>
+
 <body>
-    
+    <header>
+        <h1>J&G Weather Report</h1>
+        <p>It's always sunny in ☀️ <span id="cityName">Denver</span> ☀️</p>
+    </header>
+    <section id="city-section">
+        <p>🌇 Pick a city 🏙️ Any city 🌉</p>
+        <input type="text" id="inputCity" value="Denver">
+        <button id="cityButton">Reset</button>
+        <!-- append div from Dom??? -->
+    </section>
+    <section id="temp-section">
+        <h2>Temperature</h2>
+        <div>
+            <button id="increaseTempControl">👆</button>
+            <div id="tempValue" class='color'>70</div>
+            <button id="decreaseTempControl">👇</button>
+        </div>
+        <div>_</div>
+        <button id="realTempButton">Get Realtime Temp</button>
```

Comment:
You'll encounter many developers who dislike having mixed single and double quotes in the same file, in general in HTML many prefer double quotes.